docs/data-management/mriqc.md

```diff
@@ -1,14 +1,31 @@
 
 
-## Executing MRIQC
-- [ ] Register the MRIQC container with DataLad containers-run
-- [ ] Run MRIQC:
+## Executing *MRIQC*
+!!! info "Just once, the first time you run *MRIQC* on the dataset"
+    - [ ] Register the *MRIQC* container to the dataset
+
+        ??? info "Run another version of *MRIQC*"
+            In case you want to run another version of *MRIQC*, replace `23.1.0` with the one you intend to use in the command below
+
+        ```
+        datalad containers-add --call-fmt 'singularity exec -B {{${HOME}/tmp/}} --cleanenv {img} {cmd}' mriqc --url docker://nipreps/mriqc:23.1.0
+        ```
+        We configure the containers call to automatically mount the temporary folder, because we will store the working directory there.
+
+- [ ] Run *MRIQC*:
     ```shell
+    #Assign the variable to the last session ID
+    lastsession=01
     datalad containers-run \
-        --container-name containers/mriqc \
+        --container-name mriqc \
         --input sourcedata \
-        --output . \
-        '{inputs}' '{outputs}' participant --session lastsession -w workdir
+        --output ./derivatives/mriqc-23.1.0 \
+        "{inputs} {outputs} participant --session-id ${lastsession} -w ${HOME}/tmp/hcph-dataset/mriqc-23.1.0 --mem 40G"
+    ```
+- [ ] Push the new derivatives to the remote storage.
+    ```shell
+    datalad push --to ria-storage
+    datalad push --to origin
```

Comment:
```suggestion
    datalad push --to ria-storage
    datalad push --to origin
```

I would suggest we configure `origin` to *depend on* `ria-storage`. That way there's less room for mistakes where we forget to push to `ria-storage`. WDYT?